_layouts/redirect_external.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="refresh" content="0; URL={{ page.to }}" />
+    <title>Document</title>
+  </head>
+  <body>
+    <p>
+      This page has been moved. If you are not redirected within 3 seconds,
```

Comment:
Don't think it should say "this page has moved" -- in this case, we're trying to redirect from our page to another page.

Maybe "This page is a redirect. If you ..."?